lib/tasks_private/terraform/provision.rake

```diff
@@ -0,0 +1,91 @@
+require "json"
+require "byebug"
+require 'yaml'
+require 'logger'
+require "byebug"
+
+begin
+  require 'ruby-terraform'
+rescue LoadError
+  raise("unable to provision vm on Power VC as ruby terraform gem is not available")
+end
```

Comment:
Since this will likely be run via `rake`, which invokes bundler, it might just be easier to put this in the .gemspec as a development dependency

Reply:
Oh I see it is already in the gemspec, so I don't think you need this LoadError check at all.